exercises/tuning-resampling/ic_sol_tuning.R

```diff
@@ -0,0 +1,116 @@
+# ------------------------------------------------------------------------------
+# TUNING k-NN
+# ------------------------------------------------------------------------------
+
+library(data.table)
+library(mlr3verse)
+library(ggplot2)
+
+# DATA -------------------------------------------------------------------------
+
+# Data adapted from 
+# https://www.openml.org/search?type=data&sort=runs&status=active&qualities.NumberOfClasses=lte_1&id=41021
+
+data_baseball <- fread("baseball.csv")
```

Comment:
hier vielleicht pfad "exercises/..."

Reply:
es gilt sonst in allen "rsrc"-sachen die ungeschriebene regel, dass der ort des R-files das CWD ist. aber ich pass es an, macht schon mehr sinn 👍🏻